### PR TITLE
Card ui

### PR DESCRIPTION
**Pull request by mindfulme**
#### What's this PR do?
Adds a Card-UI component
#### Where should the reviewer start?
At Card.vue
#### How should this be manually tested?
go to localhost:8080/#/components
#### Any background context you want to provide?
No
#### What are the relevant tickets?
Card-UI component
#### Questions:
I can't find any mockup related to Card-UI component on Zeplin, does it exist?

### CODE REVIEW

File: src/components/Card/Card.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <div
+    :class="{
```
Comment (roma219) [inline src/components/Card/Card.vue]: No need for dark/light themes in components. Theme switching gonna be implemented in a tailwindcss layers - that's why we use it's varialbes for colors

File: src/components/Card/Card.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      'card-dark': theme === 'dark',
+      'card-light': theme === 'light'
+    }"
+    class="card-main"
+  >
+    <div class="title">
+      {{ title }}
```
Comment (roma219) [inline src/components/Card/Card.vue]: title should be in the header

File: src/components/Card/Card.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    title: {
+      type: String,
+      default: () => ''
```
Comment (roma219) [inline src/components/Card/Card.vue]: No need for defaul funcs in string props

File: src/components/Card/Card.vue [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+<style scoped lang="scss">
+
+.title {
+  font-size: 18px;
+}
+
+.card-header {}
+
+.card-dark {
+  background-color: config('colors.card-background-dark');
```
Comment (roma219) [inline src/components/Card/Card.vue]: just use `card-background` and when changing dark/light it is gonna change in a wailwind config layer

File: src/views/UserInfo.vue [hunk 5]
```diff
@@ -8,8 +8,8 @@
 
 <script type="text/javascript">
 import { mapGetters } from 'vuex'
-import '@icons/userIcon'
-import '@icons/arrowDown'
+// import '@icons/userIcon'
```
Comment (roma219) [inline src/views/UserInfo.vue]: why that? You probably need to run `npm run svg` before starting

**Review by roma219 — CHANGES_REQUESTED** [review]
> I can't find any mockup related to Card-UI component on Zeplin, does it exist?

Yes, check out Markets and Order book widgets - the idea is to use this Card UI component to display further widgets.

File: src/components/Card/Card.vue [hunk 6]
```diff
@@ -30,9 +30,11 @@ export default {
 .title {
   text-align: left;
   padding: 5px;
+  font-weight: 500;
 }
 
 .card {
+  font-family: Gotham_Pro_Regular;
```
Comment (roma219) [inline src/components/Card/Card.vue]: there is a tailwind variables with font families - and you won't need font-weight for that

Comment (mindfulme) [reply src/components/Card/Card.vue]: got it

**Comment by roma219** [comment]
@MindfulMe 
image
 
Should look identical to this:
image

image

And please provide screenshots in PR main message next time

**Comment by roma219** [comment]
@MindfulMe And from now on the main development branch is `dev` - so use it to branch out from and to make PRs to.

**Comment by mindfulme** [comment]
What else should be added? I just placed components that come to slot tags to the right positions. Because I guess the search box and other interesting stuff is just some components that will be passed via slots

**Comment by mindfulme** [comment]
font sizes from zeplin does not match the page sizes anyways 🙄

**Comment by roma219** [comment]
@MindfulMe paddings, bg color, title styles

**Comment by mindfulme** [comment]
also, I think I can pull&merge changes from dev and make a pull request directly to dev

**Comment by roma219** [comment]
@MindfulMe just edit this PR

**Comment by mindfulme** [comment]
okay..)

**Comment by mindfulme** [comment]
Is it okay?screen shot 2018-10-08 at 18 56 58

**Comment by MindfulMe** [comment]
This is exactly according to Zeplinscreen shot 2018-10-08 at 19 15 41

**Comment by mindfulme** [comment]
Exactly according to Zeplinscreen shot 2018-10-08 at 19 20 08

**Comment by roma219** [comment]
@MindfulMe Looks good.
merge `dev` branch and edit this PR to target `dev`:
image

**Comment by mindfulme** [comment]
yep 👍

File: tailwind.js [hunk 7]
```diff
@@ -126,6 +126,10 @@ let colors = {
   'pink-lighter': '#ffbbca',
   'pink-lightest': '#ffebef',
 
+  'text-primary': '#000000',
```
Comment (roma219) [inline tailwind.js]: it is already defined

File: tailwind.js [hunk 8]
```diff
@@ -126,6 +126,10 @@ let colors = {
...
+  'input-border': '#ced2d6',
+  'card-background': '#000000',
+  'card-text': '#ffffff',
```
Comment (roma219) [inline tailwind.js]: use 'text-primary' instead

File: tailwind.js [hunk 9]
```diff
@@ -657,7 +661,7 @@ module.exports = {
     '20': '5rem',
     '24': '6rem',
     '32': '8rem',
-
+    'card-ui': '11px',
```
Comment (roma219) [inline tailwind.js]: use rems

File: tailwind.js [hunk 10]
```diff
@@ -692,7 +696,8 @@ module.exports = {
     '16': '4rem',
     '20': '5rem',
     '24': '6rem',
-    '32': '8rem'
+    '32': '8rem',
+    'card-ui-header': '27px'
```
Comment (roma219) [inline tailwind.js]: rems

File: src/components/Card/Card.vue [hunk 11]
```diff
@@ -0,0 +1,61 @@
+<template>
+  <div
+    class="card"
+  >
+    <div class="card-header">
+      <div class="title">
+        <div> {{ upper(title) }} </div>
```
Comment (roma219) [inline src/components/Card/Card.vue]: You can do it via css

**Review by roma219 — CHANGES_REQUESTED** [review]
Small fixes

File: src/components/Card/Card.vue (hunk 11 above)
Comment (mindfulme) [reply src/components/Card/Card.vue]: got it

**Comment by roma219** [comment]
@MindfulMe next time write use number of issue in branchname and in PR name. Also you can reference it like "closes #52" in PR message to automatically close it when PR is accepted.

**Review by roma219 — APPROVED** [review]

**Pull request merged**